xchange-kraken/src/test/java/org/knowm/xchange/kraken/KrakenUtilsTest.java

```diff
@@ -66,6 +67,7 @@ public void testAdaptTradeHistoryByCurrencyPair()
 
     UserTrades userTrades = KrakenAdapters.adaptTradesHistory(filteredKrakenTradeMap);
 
+    userTrades = new UserTrades(userTrades.getUserTrades(), TradeSortType.SortByID);
```

Comment:
Maybe combine these two lines? Either way looks a bit hacky but I can understand that you want to avoid changing the main code.

Reply:
Incorporated into actual PR

Reply:
https://github.com/knowm/XChange/pull/4788